### DISCUSSION TITLE
Add more control over assigning roles/jobs

### DISCUSSION THREAD

**Discussion by StruggleDuck** [FakeFishGames/Barotrauma#7146] [Ideas and suggestions]
**TLDR; Server admins should be given the ability to control how roles/jobs are assigned, as well as assigning specific people a specific role.**

 Essentially, what I think should be added is that there should be options for admins to control how the game distributes roles, by setting certain variables, such as how many players are required to be a specific role, or the maximum amount of people that are allowed to have a certain role.

 The other thing that should be added is the ability to manually assign people roles, which would guarantee that they get that role. This would effectively solve the issue of crews being completely unbalanced role wise, as well as solving the issue of people not getting the roles they chose, even if they are they admins on the server. This could also be used to force more untrustworthy crewmates to not have access to greater tools for griefing that they would have access to by being certain roles.
